services/net/syndication/SyndicationAction.cs

```diff
@@ -80,6 +84,26 @@ private async Task ImportFeedAsync(IDataSourceIngestManager dataSource, Syndicat
         {
             try
             {
+                // Fetch content body separately if required
+                if (bool.Parse(dataSource.DataSource.GetConnectionValue("fetchContent")))
+                {
+                    var link = item.Links.FirstOrDefault(l => l.RelationshipType == "alternate")?.Uri;
+                    if (Uri.IsWellFormedUriString(link.ToString(), UriKind.Absolute))
+                    {
+                        var content =  await this.GetContentAsync(link);
+                        var date = await GetPubDateTimeAsync(content, dataSource.DataSource);
+
+                        item.Id = link.ToString();
+                        item.PublishDate = date;
+                        content = StringExtensions.SanitizeContent(content);
```

Comment:
FYI - Extensions can also be used the following way `content.SantizeContent()`, but isn't required.